docs/source/developers/release.rst

```diff
@@ -487,7 +487,39 @@ Be sure to go through on the following checklist:
    :class-title: sd-fs-5
    :class-container: sd-shadow-md
 
-   TODO: Move from https://cwiki.apache.org/confluence/display/ARROW/Release+Management+Guide#ReleaseManagementGuide-UpdatingRpackages
+   To publish the R package on CRAN, there are a few steps we need to do first
+   in order to ensure that binaries for Windows and macOS are available to CRAN.
+   Jeroen Ooms <jeroenooms@gmail.com> maintains several projects that build C++
+   dependencies for R packages for macOS and Windows. We test copies of these
+   same build scripts in our CI, and at release time, we need to send any
+   changes we have and update the versions/hashes upstream.
+
+   When the release candidate is made, make draft pull requests to each
+   repository using the rc, updating the version and SHA, as well as any cmake
+   build changes from the corresponding files in apache/arrow. Jeroen may
+   merge these PRs before the release vote passes, build the binary artifacts,
+   and publish them in the right places so that we can do pre-submission checks
+   (see below). After the release candidate vote passes, update these PRs
+   to point to the official (non-rc) URL and mark them as ready for review.
```

Comment:
Where would I find this official URL?

Reply:
Each of these scripts has the official and rc URLs in them, and it's a matter of updating the versions and (un)commenting to enable one or the other.